ddtrace/internal/safety.py

```diff
@@ -66,13 +67,15 @@ def __call__(self, *args, **kwargs):
 
     def __getattribute__(self, name):
         # type: (str) -> Any
-        if name == "__wrapped__":
+        if PYTHON_VERSION_INFO[0:2] < (3, 12) and name == "__wrapped__":
```

Comment:
Interesting 🤔. Why doesn't this work with CPython 3.12?
```suggestion
        if PYTHON_VERSION_INFO < (3, 12) and name == "__wrapped__":
```

Reply:
To be honest I'm not certain. I suspect it has to do with this change in the latest version of wrapt https://github.com/GrahamDumpleton/wrapt/commit/dd81bd7b5b604c58572c4863c25bdd2385d56468